src/CUDAdrv.jl

```diff
@@ -5,17 +5,15 @@ module CUDAdrv
 using Compat
 using Compat.String
 
-const ext = joinpath(dirname(@__DIR__), "deps", "ext.jl")
-if isfile(ext)
-    include(ext)
-elseif haskey(ENV, "ONLY_LOAD")
+include("../deps/ext.jl")
```

Comment:
Hmm, Julia should really be normalising these in some way so that's not necessary